src/utils/__tests__/csvColumnAnalysis.test.ts

```
@@ -252,6 +292,9 @@ describe('csvColumnAnalysis', () => {
     it('analyzes CSV file headers', async () => {
       const csvContent = 'title,status,company\n"Test Project","completed","Test Company"';
       const file = new File([csvContent], 'test.csv', { type: 'text/csv' });
+
+      const result = await analyzeCSVFile(file);
+
       
       const result = await analyzeCSVFile(file);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'result'. Consider to delete it or rename it.

'result' is defined here:


(lint/suspicious/noRedeclare)